src/lib/components/ui/Modal/Modal.jsx

```diff
@@ -142,6 +143,7 @@ Modal.propTypes = {
     id: PropTypes.string,
     label: PropTypes.string.isRequired,
     loadingIcon: PropTypes.element,
+    type: PropTypes.string,
```

Comment:
I think we should provide whitelist like we do here: https://github.com/react-ui-org/react-ui/blob/f949df4fef3f6aa611190e7fd431670098fc5428/src/lib/components/ui/Button/Button.jsx#L139